src/channel.rs

```diff
@@ -1,73 +1,1000 @@
+    ///     .cloud(None)
+    ///     .link("http://www.jupiterbroadcasting.com/")
+    ///     .finalize()
+    ///     .unwrap();
+    ///
+    /// assert!(channels.cloud().is_none());
+    /// ```
+    pub fn cloud(&self) -> Option<Cloud>
+    {
+        self.cloud
+            .clone()
+    }
+
+
+    /// Get the optional ttl that exists under `Channel`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use rss::{ChannelBuilder, Channel};
+    ///
+    /// let ttl_num = 60;
+    ///
+    /// let channels = ChannelBuilder::new()
+    ///     .ttl(Some(ttl_num))
+    ///     .link("http://www.jupiterbroadcasting.com/")
+    ///     .finalize()
+    ///     .unwrap();
+    ///
+    /// let ttl_option = channels.ttl();
+    /// assert!(ttl_option.is_some());
+    ///
+    /// assert_eq!(ttl_num.to_string(), ttl_option.unwrap());
+    /// ```
+    ///
+    /// ```
+    /// use rss::{ChannelBuilder, Channel};
+    ///
+    /// let channels = ChannelBuilder::new()
+    ///     .ttl(None)
+    ///     .link("http://www.jupiterbroadcasting.com/")
+    ///     .finalize()
+    ///     .unwrap();
+    ///
+    /// assert!(channels.ttl().is_none());
+    /// ```
+    pub fn ttl(&self) -> Option<String>
+    {
+        self.ttl
+            .clone()
+    }
+
+
+    /// Get the optional image that exists under `Channel`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use rss::{ChannelBuilder, Channel, ImageBuilder};
+    ///
+    /// let image = ImageBuilder::new()
+    ///     .link("http://www.jupiterbroadcasting.com")
+    ///     .url("http://jupiterbroadcasting.com/images/LAS-300-Badge.jpg")
+    ///     .title("LAS 300 Logo")
+    ///     .height(None)
+    ///     .width(None)
+    ///     .description(None)
+    ///     .finalize()
+    ///     .unwrap();
+    ///
+    /// let channels = ChannelBuilder::new()
+    ///     .image(Some(image))
+    ///     .link("http://www.jupiterbroadcasting.com/")
+    ///     .finalize()
+    ///     .unwrap();
+    ///
+    /// assert!(channels.image().is_some());
+    /// ```
+    ///
+    /// ```
+    /// use rss::{ChannelBuilder, Channel};
+    ///
+    /// let channels = ChannelBuilder::new()
+    ///     .image(None)
+    ///     .link("http://www.jupiterbroadcasting.com/")
+    ///     .finalize()
+    ///     .unwrap();
+    ///
+    /// assert!(channels.image().is_none());
+    /// ```
+    pub fn image(&self) -> Option<Image>
+    {
+        self.image
+            .clone()
+    }
+
+
+    /// Get the optional rating that exists under `Channel`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use rss::{ChannelBuilder, Channel};
+    ///
+    /// let channels = ChannelBuilder::new()
+    ///     .rating(None)
+    ///     .link("http://www.jupiterbroadcasting.com/")
+    ///     .finalize()
+    ///     .unwrap();
+    ///
+    /// assert!(channels.rating().is_none());
+    /// ```
+    pub fn rating(&self) -> Option<String>
+    {
+        None
```

Comment:
It doesn't look like there's a `rating` property on `Channel`. That should be added and this should return `self.rating`.